src/libraries/Microsoft.Extensions.DependencyInjection/tests/Microsoft.Extensions.DependencyInjection.Tests.csproj

```diff
@@ -2,18 +2,21 @@
 
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetFrameworkCurrent)</TargetFrameworks>
```

Comment:
These tests will use the `netstandard2.0` implementation, do we want to add a `NetFrameworkCurrent` TFM to the source project and exclude it from the package?

Reply:
Actually they try to test the `net461` build but if fails to build because of missing references.  We need to exclusion.